lein-light-nrepl/src/lighttable/nrepl/eval.clj

```diff
@@ -70,7 +81,9 @@
      (and (string? res) (:verbatim opts)) res
      :else (pr-str res))))
 
-(defn truncate [v]
+(defn truncate
+  "Useless function... Same as 'identity'. Deprecated and to be removed."
```

Comment:
Worth removing as this is needless commentary on a fn that is self explanatory. Also, dev comments/todos don't belong in fn docstrings